src/assets/signals/daa.svg

```diff
@@ -1 +1 @@
-<svg width="32" height="33" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M2 4h17v27H2V4z" fill="#E7EAF3"/><path d="M12 2h18v12H12V2z" fill="#fff"/><path d="M14.75 10.5a.75.75 0 0 1 .75-.75h3a.75.75 0 0 1 0 1.5h-3a.75.75 0 0 1-.75-.75zM25.5 9.75a.75.75 0 0 0 0 1.5h1a.75.75 0 0 0 0-1.5h-1z" fill="#181B2B"/><path fill-rule="evenodd" clip-rule="evenodd" d="M10.5 25.75a1.75 1.75 0 1 0 0 3.5 1.75 1.75 0 0 0 0-3.5zm-.25 1.75a.25.25 0 1 1 .5 0 .25.25 0 0 1-.5 0z" fill="#181B2B"/><path fill-rule="evenodd" clip-rule="evenodd" d="M10.75 1.5a.75.75 0 0 1 .75-.75h19a.75.75 0 0 1 .75.75v13a.75.75 0 0 1-.75.75h-19a.75.75 0 0 1-.75-.75V9.25h-8.5v13.5h16.5V18.5a.75.75 0 0 1 1.5 0v11a2.756 2.756 0 0 1-2.75 2.75h-14A2.756 2.756 0 0 1 .75 29.5v-24A2.756 2.756 0 0 1 3.5 2.75h7.25V1.5zm19 .75h-17.5v2.5h17.5v-2.5zm-17.5 4h17.5v7.5h-17.5v-7.5zm-1.5-2H3.5c-.686 0-1.25.564-1.25 1.25v2.25h8.5v-3.5zm-8.5 20v5.25c0 .686.564 1.25 1.25 1.25h14c.686 0 1.25-.564 1.25-1.25v-5.25H2.25z" fill="#181B2B"/></svg>
\ No newline at end of file
+<svg width="28" height="30" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#clip0)"><path d="M14.123 28.167c-3.338 1.764-6.877-3.162-10.154-5.798C.69 19.733-.61 15.524 2.647 10.729c3.256-4.795 6.519.958 8.943 1.672 2.424.714 7.63-1.757 9.814 0 2.185 1.757.687 7.348-2.704 7.091-3.392-.256-.406 6.47-4.577 8.675z" fill="#E7EAF3"/><circle cx="10.008" cy="12.8" r="7.663" stroke="#505573" stroke-width="1.5"/><path fill-rule="evenodd" clip-rule="evenodd" d="M18.377 13.66a8.498 8.498 0 0 0 .019-1.5h6.823v-1.118a2.5 2.5 0 0 0-2.5-2.5h-5.455a8.436 8.436 0 0 0-1.123-1.5h6.578a4 4 0 0 1 4 4V21.767a4 4 0 0 1-4 4H8.249a4 4 0 0 1-4-4v-2.833c.455.427.958.805 1.5 1.123v1.71a2.5 2.5 0 0 0 2.5 2.5h14.47a2.5 2.5 0 0 0 2.5-2.5v-8.108h-6.842z" fill="#505573"/><path fill-rule="evenodd" clip-rule="evenodd" d="M14.575 10.444c0 .238-.112.45-.285.588l-2.24 1.793a.75.75 0 0 1-.938-1.172l.575-.46H6.195a.75.75 0 1 1 0-1.5h5.492l-.575-.459a.75.75 0 0 1 .937-1.17l2.237 1.789a.749.749 0 0 1 .289.59zM5.73 14.566a.748.748 0 0 0-.004 1.18l2.242 1.792a.75.75 0 0 0 .937-1.172l-.575-.46h5.492a.75.75 0 0 0 0-1.5H8.33l.574-.459a.75.75 0 0 0-.937-1.171l-2.237 1.79z" fill="#505573"/><path d="M19.23 21.242h3.744" stroke="#505573" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h28v30H0z"/></clipPath></defs></svg>
```

Comment:
hm, how active svg could be green on screenshot, if you don't use it as a component?

Reply:
Will change to component(without imports of svg)